exonum-java-binding-core/rust/ejb-app/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.4.0-SNAPSHOT"
 authors = ["Exonum team <exonum@bitfury.com>"]
 
 [dependencies]
-java_bindings = { path = "..", features = ["invocation"] }
+java_bindings = { path = "..", features = ["invocation", "resource-manager"] }
```

Comment:
@skletsun Would you please document the reason we discussed why it shall be enabled in the application to avoid confusion (and consider improving documentation of the feature, if it makes sense)?